datasetup/build_US_setup.R

```diff
@@ -7,11 +7,10 @@
 # ## Configuration Items
 #
 # ```yaml
-# data_path: <path to directory>
 # subpop_setup:
 #   modeled_states: <list of state postal codes> e.g. MD, CA, NY
-#   mobility: <path to file relative to data_path> optional; default is 'mobility.csv'
-#   geodata: <path to file relative to data_path> optional; default is 'geodata.csv'
+#   mobility: <path to file relative> optional; default is 'mobility.csv'
```

Comment:
relative to ... what?

I think unfortunately we don't have a super-clear answer to that. For the time being, maybe more like "path to a file; may be absolute or relative" - and leave specification of how relative works to future updates.

Reply:
(also applies to next line)